addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -152,8 +152,17 @@ options.registry.carousel = options.Class.extend({
     onClone: function () {
         var id = 'myCarousel' + new Date().getTime();
         this.$target.attr('id', id);
-        this.$target.find('[data-slide]').attr('href', '#' + id);
-        this.$target.find('[data-slide-to]').attr('data-target', '#' + id);
+        const $dataSlide = this.$target.find('[data-slide]');
+        const $dataSlideTo = this.$target.find('[data-slide-to]');
+        const setTargetId = $el => {
+            if ($el.attr('data-target')) {
+                $el.attr('data-target', '#' + id);
+            } else if ($el.attr('href')) {
+                $el.attr('href', '#' + id);
+            }
+        };
+        setTargetId($dataSlide);
+        setTargetId($dataSlideTo);
```

Comment:
Don't know if I should have seen this earlier, but you cannot use ES6 syntax before 13.0, so no "const", arrowed functions, etc.

Reply:
Also you should use a named function anyway, no set it in a variable.

At the end of the `onClone` function, define `function xxx() {` then you can use it anywhere in `onClone`.

Reply:
done :)